fixtures/dom/src/components/fixtures/textareas/index.js

```diff
@@ -10,7 +10,7 @@ export default class TextAreaFixtures extends React.Component {
   };
   render() {
     return (
-      <FixtureSet title="Textareas">
+      <FixtureSet title="Textareas" description="">
```

Comment:
Really, I think `description` shouldn't be a required PropType on FixtureSet. Do you mind changing that instead?

Reply:
Sure, I'll change it. I found it odd as well initially, but figured I'd be better off leaving it. 🤷